src/datacatalog/config_schema.yml

```diff
@@ -55,6 +54,17 @@ definitions:
   jwks:
     type: string
 
+  jwks_url:
+    type: string
+
+  min_interval_keyset_update:
+    type: integer
+
+  allowed_signing_algorithms:
+    type: array
+    items:
+      type: string
```

Comment:
Deze properties moeten nog worden toegevoegd aan het `#/properties` object in regel 5:
```yaml
properties:
  ...
  jwks_url:
    $ref: '#/definitions/jwks_url'
  min_interval_keyset_update:
    $ref: '#/definitions/min_interval_keyset_update'
  allowed_signing_algorithms:
    $ref: '#/definitions/allowed_signing_algorithms'
  ...
```

Reply:
done